### PR TITLE
Added contributors to the list

### PR DESCRIPTION
**Pull request by Adriani90**
### Link to issue number:
contributes to #9725. 

### Summary of the issue:
Some long term contributors are not yet in the contributors list and they asked for that on the translations maillist. Also many other contributors which are very active on users lists, addons maillist etc. are still not on the list.

### Description of how this pull request fixes the issue:
added the names to the contributors.txt 

### Testing performed:
Ran NVDA from local branch and checked that the list is updated

### Known issues with pull request:
none

### Change log entry:
== changes ==
- Updated the list of contributors.

### CODE REVIEW

**Comment by Andre9642** [comment]
I'd suggest to add the following link to the contributors.txt: <https://github.com/nvaccess/nvda/graphs/contributors>. Indeed, several gitHub contributors are not listed in `contributors.txt`, like:

* Christopher Toth
* Bill Dengler
* Aron Ocsvari
* Josiel Santos
* Karol Pecyna
* Blake Oliver
* Robert Haenggi
* Mikolaj Holysz
* André-Abush Clause
* Etc.

**Comment by Adriani90** [comment]
@Andre9642 I have added the link to the list along with a short description at the top of the list and added the names mentioned by you at the bottom of the list. In my view, I think we will never be able to include really everyone to the list because we do not know really how many people contribute to NVDA out there that we don't know of. So I think the best way is to raise such a pull request from time to time and add new names. As of now I can add as many names as possible. If someone knows further people who should be included on this list, please comment on this pull request.

**Comment by michaelDCurran** [comment]
I will merge this pr to beta this Friday after 0 UTC. This gives a few more days for people to get their names added if missing. Then, they will be included in rc1.

File: contributors.txt
Comment (lukaszgo1) [inline contributors.txt]: I don't believe 'either way' is a proper phrase here because it implies that it is possible to contribute only in two ways.

**Comment by josephsl** [comment]
Hi, I think a good word for this could be “various”. Thanks.

**Comment by lukaszgo1** [comment]
I don't know if they are any strict requirements  for being placed in the contributors file, but if not and my contributions are enough I'd like to ask to be added into it.

**Comment by Adriani90** [comment]
Lukasz ofcourse in my view you contribute alot to this project. I have added your name as well.

**Comment by XLTechie** [comment]
Why is any phrase with "way" in it necessary? Why not just:

"
This is a list of people and organizations which have contributed to the NVDA 
project since the project has started. For an overview of code contributors, see 
also
"

I would also suggest changing "which" to "who"; and "has started" to "began" or 
at least just "started", but those were not what was asked.:)

**Comment by Adriani90** [comment]
I decided to add also a line in changes.t2t so people are aware that there has been an update to the list. @michaelDCurran as of now the commit for changes is for 2019.3. This is just one string. Should I amend it and commit it to your RC branch?

**Comment by lukaszgo1** [comment]
I'm afraid change log entry isn't a good idea here because it would be merged to beta directly, and the addition to change log wouldn't be translated.

**Comment by Adriani90** [comment]
Not necessarily, in case Mick decides to release a 2019.2.1 this could be translated. Or am I wrong?
Anyway, in my view it is important to let people know that there is a list of contributors and that it is updated from time to time. If there will not be a 2019.2.1, then I suggest to implement this in 2019.3.

**Review by josephsl — CHANGES_REQUESTED** [review]
Hi,

I would like to request not editing What's New document in the future. Although mentioning new contributors has its merits, it isn't necessary and could open up issues requesting trivial changes be documented. For these, git log would suffice for now.

Thanks.

**Comment by Adriani90** [comment]
Oh yes I just discovered editing changes is not a really good idea :-). I have reverted the coresponding commit.

**Review by michaelDCurran — APPROVED** [review]

**Comment by michaelDCurran** [comment]
I have rebased the branch and retargeted pr the to beta before merging.

**Pull request merged**
